### PR TITLE
Automatic screenshots via GitHub Actions.

### PR DESCRIPTION
**Pull request by dennykorsukewitz**
Discussions: https://github.com/karldaeubel/PenAndPaperFloorplanner/discussions/27

- Added .github/workflows/screenshot.yml
- Added .github/workflows/pages.yml
- Added js/example.js
- Added loadRemoteExample() in io.js
- Added and refactored loadFloorPlan() function

---

# Todo's

- [x] Add `paths` to worfklow Screenshot https://github.com/karldaeubel/PenAndPaperFloorplanner/pull/28/files#diff-1dff6a1148a585064cd8019c48b00f466d219516e7802a4ae06db7935ae78f27R4-R8

---

- [x] Don't forget that the action needs more rights to push on your behalf.

Bildschirmfoto 2023-09-21 um 20 45 34

---

- [x] Change Source from "Deploy form a Branch" to "GitHub Actions"

Bildschirmfoto 2023-09-28 um 14 53 19

### CODE REVIEW

**Comment by dennykorsukewitz** [comment]
@karldaeubel 
From my side, it looks good.

Maybe we should take another look at your Pages deployment.
We should build an action from it and after the deployment the screenshot job should run.
We also have to make sure that they don't trigger each other. LOOP in coming 😄

**Comment by karldaeubel** [comment]
The question is does this job trigger itself? Because it is commiting new pictures. Or is it triggered by the pages axtion? It should run like 5 Minutes or so after something was committed, but without triggering itself. How did that work on your branch?

File: js/example.js
[the remark this answers is not in the document]
Comment (karldaeubel) [reply js/example.js:1]: Done

**Review by karldaeubel — APPROVED** [review]
Done.

**Pull request merged**